lib/backend/auth/authService.dart

```diff
@@ -38,50 +38,31 @@ class AuthService {
 
     // If the user is authenticated, start the phone number verification process
     if (userCredential != null && userCredential.user != null) {
-      await FirebaseAuth.instance.verifyPhoneNumber(
-        phoneNumber: phoneNumber,
-        verificationCompleted: (PhoneAuthCredential credential) async {
-          await FirebaseAuth.instance.signInWithCredential(credential);
-          Navigator.push(
-            context,
-            MaterialPageRoute(
-              builder: (context) => const DashboardPage(),
-            ),
-          );
-        },
-        verificationFailed: (FirebaseAuthException e) {
-          showDialogBox(
-            context,
-            Icons.error,
-            Colors.red,
-            Colors.red,
-            'Error',
-            e.message ?? 'An error occurred.',
-            () => Navigator.of(context).pop(),
-          );
-        },
-        codeSent: (String verificationId, int? resendToken) {
-          Navigator.push(
+      myauth.setConfig(
+          appEmail: "99marafay@gmail.com",
+          appName: "Admin",
+          userEmail: email,
+          otpLength: 4,
+          otpType: OTPType.digitsOnly);
+
+      var template =
+          'Hi {{app_name}} Your OTP Code For Admin Dashboard is {{otp}}.';
+      myauth.setTemplate(render: template);
```

Comment:
okay

Reply:
The issue is that I cannot use a .env file because it is not suitable for production. Our production environment is fully automated, with auto-build and auto-deploy using different services. Therefore, I am leaving it as it is.